app.yaml

```diff
@@ -25,6 +25,9 @@ handlers:
 - url: /static
   static_dir: static
   secure: always
+- url: /latest/.*
+  script: _go_app
+  secure: always
```

Comment:
Adding the route to app.yaml isn't necessary, it'll be caught in the router in main.go.

Reply:
Done.